app/index.html

```diff
@@ -50,7 +50,7 @@
                                     </strong>
                                 </span>
                                 <span class="block m-t-xs">
-                                    <strong class="font-bold">Звання</strong>
+                                    <strong class="font-bold">{{vm.ranksNames[vm.currentUser.userRank]}}</strong>
```

Comment:
The same here. All logic should be moved to Cotroller, and on view we use methods from this controllers.